graphdatascience/session/gds_sessions.py

```diff
@@ -32,13 +32,12 @@ class SessionInfo:
 
     @classmethod
     def from_specific_instance_details(cls, instance_details: InstanceSpecificDetails) -> SessionInfo:
-
         size = ""
         try:
             # instance creation also allows for GB but instance details returns size as GiB
-            memory = instance_details.memory.replace("GiB", "GB").replace(" ", "")
+            memory = instance_details.memory
             if memory:
-                size = SessionSizeByMemory(memory).name
```

Comment:
do you want to change the SessionInfo to also say `memory` instead of size?

Reply:
Yes, good catch.